astropath/tests/test_priors.py

```diff
@@ -38,3 +38,22 @@ def test_renorm_priors():
     # U != 0
     renorm_Oi = priors.renorm_priors(raw_Oi, 0.1)
     assert np.all(renorm_Oi == np.array([0.1125, 0.225, 0.5625]))
+
+def test_norm_theta_priors():
+    # Grid me
+    ngrid = 2000
+    x = np.linspace(-30., 30., ngrid)
+    grid_spacing_arcsec = x[1]-x[0]
+
+    xcoord, ycoord = np.meshgrid(x,x)
+    theta = np.sqrt(xcoord**2 + ycoord**2)
+
+    # Uniform
+    for pdf in ['core', 'uniform', 'exp']:
+        theta_prior = dict(max=6, PDF=pdf, scale=1.)
```

Comment:
This is fine, though for tests, I'd try to avoid using scale=1. and similar, since setting things to 1 can hide sensitivities. Not terribly important however.